src/Average.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param None
```

Comment:
This `@param` annotation can be omitted entirely unless defining the overload which takes a function here.

Reply:
I implemented the overload case and fixed this annotation to correctly match that overload.